.eslintrc.js

```diff
@@ -11,6 +11,7 @@ module.exports = {
     },
     "extends": [
         "eslint:recommended",
+        "plugin:compat/recommended",
```

Comment:
@unlikelyzero It looks like this is causing a lint failure due to:
```
/root/project/src/plugins/notebook/components/Notebook.vue
  441:32  error  screen.orientation() is not supported in iOS Safari 15.2-15.3  compat/compat
```
I remember you and @akhenry talking about this. Sure enough, it [doesn't appear to be supported by iOS Safari](https://developer.mozilla.org/en-US/docs/Web/API/Screen/orientation). It looks like `window.orientation`, [despite being deprecated](https://developer.mozilla.org/en-US/docs/Web/API/Window/orientation), works [in iOS Safari](https://stackoverflow.com/a/70882036).

Should add an [eslint ignore](https://eslint.org/docs/user-guide/configuring/ignoring-code) and file a ticket to fix this in the code?

Reply:
@scottbell yes! We have a Github Project for 'Mobile'. If you don't mind filing the follow up ticket with that Project, it'll be easier to trace

Reply:
Ah, @akhenry had already filed a bug with the exact wording I would have used :)
https://github.com/nasa/openmct/issues/4875
I'll add it to the Mobile project